utils/injectionDependances/container.ts

```diff
@@ -1,25 +1,30 @@
 import { ActionsService } from 'services/actions.service'
+import { JeunesService } from '../../services/jeunes.service'
+import { RendezVousService } from '../../services/rendez-vous.service'
 
 export interface Dependances {
```

Comment:
```suggestion
export interface Dependencies {
```
nitpick: on a un mix entre anglais et français